api/src/main/java/com/codeforcommunity/rest/subrouter/authenticated/ProtectedSchoolRouter.java

```diff
@@ -239,6 +248,13 @@ private void handleCreateReportWithLibrary(RoutingContext ctx) {
     end(ctx.response(), 201, JsonObject.mapFrom(report).toString());
   }
 
+  private void handleGetMostRecentReport(RoutingContext ctx) {
+    JWTData userData = ctx.get("jwt_data");
+    int schoolId = RestFunctions.getPathParamAsInt(ctx, "school_id");
+    ReportGeneric report = processor.getMostRecentReport(userData, schoolId);
```

Comment:
Instead of having the report be of type ```ReportGeneric```, for consistency can you make the report be of the type of ```MostRecentReport``` or whatever you called it.

Reply:
I think `ReportGeneric` is okay, since that DTO actually doubles for paginated reports, so it's technically not always the most recent, and there's no reason to duplicate the DTO just for the name of the class.